cirq/protocols/circuit_diagram_info_protocol_test.py

```diff
@@ -183,7 +183,7 @@ def test_circuit_diagram_info_args_repr():
 
 
 def test_format_real():
-    args = cirq.CircuitDiagramInfoArgs.UNINFORMED_DEFAULT.copy()
```

Comment:
Why remove copy()? Otherwise the same instance is going to be mutated across multiple tests that can result in surprising errors.